package.json

```diff
@@ -41,7 +41,7 @@
     "glob": "^7.1.1",
     "graceful-fs": "^4.1.11",
     "immutable": "^4.0.0-rc.7",
-    "istanbul-api": "^1.1.0",
+    "istanbul-api": "1.3.1",
```

Comment:
Please don't lock it

Reply:
The version here doesn't matter either, as it's not published. Update should go here: https://github.com/facebook/jest/blob/df34b204c3aa8dfc3a05d19039c6d538cbce9135/packages/jest-cli/package.json

Reply:
ok, will update my MR, thank you